components/channel_header_mobile/channel_info_button/__snapshots__/channel_info_button.test.js.snap

```diff
@@ -133,10 +135,13 @@ exports[`components/ChannelHeaderMobile/ChannelInfoButton should match snapshot,
             <FormattedMessage
               defaultMessage="Info Icon"
               id="generic_icons.info"
-              values={Object {}}
             >
               <svg
-                aria-label="Info Icon"
+                aria-label={
```

Comment:
Is it okay that we're passing an array of strings into `aria-label`?

Reply:
No. Sigh... all the `../widgets/icons/**.jsx` svg icons are use the child func syntax.
```jsx
<FormattedMessage>
    {(ariaLabel) => (
        <svg>
    )}
</FormattedMessage>
```

I'll get those all changed over.

Reply:
Ah, that's why `wrapRichTextChunksInFragment` isn't default. That answers that question then.

Thanks for changing all those. I'm also realizing we should make it so that `make i18n-extract` can recognize `useIntl` when looking for translation strings so that we don't need to `t` when using it

Reply:
There's at least another ~25 instances of the FormattedMessage func child pattern that could be hiding issues. I found one that will need to be resolved before this is ready.

That'd be great! (`i18n-extract`/`useIntl`/`t()`)